src/public/js/at-sce.util.js

```diff
@@ -42,6 +42,81 @@ $(function () {
 });
 
 /* Calculator Functions */
+function SumFixedCost() {
+  var rent = document.getElementById("rent").value;
+  var transport = document.getElementById("transport").value;
+  var telephone = document.getElementById("telephone").value;
+  var feeding = document.getElementById("feeding").value;
+  var others = document.getElementById("others").value;
+  var totalFixedCost =
+    parseFloat(rent) +
+    parseFloat(transport) +
+    parseFloat(telephone) +
+    parseFloat(feeding) +
+    parseFloat(others);
+  return totalFixedCost;
+}
+
+function CostPerDay() {
+  var days = document.getElementById("days").value;
+  var costDay = SumFixedCost() / days;
+  return costDay;
+}
+
+function CostPerHour() {
+  var hours = document.getElementById("hours").value;
+  var costHour = CostPerDay() / hours;
+  return costHour;
+}
+
+function TaxesIVAandISR() {
```

Comment:
Read about the single responsibility principle and best practices for functions https://hackernoon.com/understanding-solid-principles-in-javascript-w1cx3yrv